node/tools/src/config.rs

```diff
@@ -118,6 +118,13 @@ impl ProtoFmt for AppConfig {
     }
 }
 
+impl AppConfig {
+    pub fn add_seed_peers(&mut self, seed_peers: Vec<(std::net::SocketAddr, node::PublicKey)>) {
```

Comment:
from the pov of the config, seed peers do not differ at all from regular peers afaiu. No special method is required and you can just use HashMap::extend method to insert these. Instead of "seed" for a flag name use the name of the field that you are appending to, and prefix it with "append" or sth to indicate that the flag is for appending rather than overriding for example. keep the (key,ip) order (rather than (ip,key) in the flag spec to be consistent with the config format.

Reply:
👍 Makes sense